packages/main-ui/src/pages/members/_id/controller.rs

```diff
@@ -72,7 +72,7 @@ impl Controller {
     ) -> Self {
         let translates: MemberDetailTranslate = translate(&lang);
         let api: MemberApi = use_context();
-        let member_resource: Resource<Result<GroupMemberRelationship, ServerFnError>> =
+        let member_resource: Resource<Result<MemberSummary, ServerFnError>> =
```

Comment:
```suggestion
        let member_resource: Resource<Result<QueryResponse<MemberSummary>>> =
```
일반적으로는 이게 될텐데 왜 Summary 가 단일로 저장되고, 에러는 ServerFnError 타입인가요?

Reply:
아직 프론트엔드 단에서는 models::Result랑 QueryResponse를 안쓰는것으로 알고있습니다